### ISSUE TITLE
Support custom WP_CONTENT_DIR locations

### ISSUE THREAD

**Issue by kasparsd** [kasparsd/minit#93]

**Comment by szepeviktor** [comment]
@markoheijnen wrote
```php
		$full_path = str_replace( WPMU_PLUGIN_URL, WPMU_PLUGIN_DIR, $url );
		$full_path = str_replace( plugins_url(), WP_PLUGIN_DIR, $full_path );
		$full_path = str_replace( get_theme_root_uri(), get_theme_root(), $full_path );
		$full_path = str_replace( content_url(), WP_CONTENT_DIR, $full_path );
```

**Comment by kasparsd** [comment]
I hope we can find a more simple way of doing the same thing. This would need a lot of inline comments to understand what it actually does :)

**Comment by szepeviktor** [comment]
1. Replace **MU plugin** URL-s
1. Replace **plugin** URL-s
1. Replace **theme** URL-s
1. Replace **remaining /wp-content** URL-s

**Comment by szepeviktor** [comment]
@kasparsd May I advance the solution for this?
Please advise.

**Comment by szepeviktor** [comment]
I'd favor a solution which moves local path determination to a **filter**.
https://github.com/kasparsd/minit/blob/master/include/minit-assets.php#L105-L110

Would you merge that?

*Then I would hook into that.*

**Comment by kasparsd** [comment]
@szepeviktor How about filtering the output of `get_asset_relative_path()` instead? It would have to return the part after `ABSPATH`.

**Comment by szepeviktor** [comment]
The base of the problem is that my wp-content is *not* below ABSPATH, they are sibling directories.
```
wp-cli.yml
$DOCROOT/─┬─index.php (modified)
          ├─wp-config.php
          ├─wp-login.php (trap)
          ├─xmlrpc.php (trap)
          ├─$CORE/─┬─index.php
          │        ├─wp-load.php
          │        ├─wp-login.php
          │        ├─wp-admin/
          │        └─wp-includes/
          └─static/ (wp-content)
```

**Comment by szepeviktor** [comment]
@kasparsd Please comment on #95

**Comment by szepeviktor** [comment]
@kasparsd Please close this one.

**Comment by kasparsd** [comment]
Fixed in #95
